### PR TITLE
Include SET-SYMBOLIC module

### PR DESCRIPTION
**Pull request by anvacaru**
Branchings with `VV0_addr_123:Int in .Set` take place. Imported the `SET-SYMBOLIC` module, which has simplifications for the Set sort.

### CODE REVIEW

File: tests/foundry/test/AccountParamsTest.t.sol [hunk 1]
```diff
@@ -38,6 +38,15 @@ contract AccountParamsTest is Test {
        assert(nonce == 1);
     }
 
+    function test_getNonce_unknownSymbolic(address addr) public {
+      vm.assume(addr != address(vm));
+      vm.assume(addr != address(this));
+      vm.assume(addr != address(0x3fAB184622Dc19b6109349B94811493BF2a45362));
+      vm.assume(addr != address(0x4e59b44847b379578588920cA78FbF26c0B4956C));
+      uint64 nonce = vm.getNonce(addr);
+      assert(nonce == 0);
```
Comment (PetarMax) [inline tests/foundry/test/AccountParamsTest.t.sol:47]: @anvacaru I think we had a discussion about this and whether it should be symbolic or not - what is the current idea?

**Comment by ehildenb** [comment]
My guess after 2 failures in a row we need to actually inspect the proofs that are taking longer. Maybe we've introduced a looping simplification.

**Comment by anvacaru** [comment]
Locally, all the proofs passed in 75 minutes. I'll look at the simplifications.
image

File: tests/foundry/test/AccountParamsTest.t.sol (hunk 1 above)
Comment (anvacaru) [reply tests/foundry/test/AccountParamsTest.t.sol:47]: @PetarMax in this PR I just want to add a test that shows the need of the `SET-SYMBOLIC` module. I've created [this issue](https://github.com/runtimeverification/evm-semantics/issues/1689) to track wether it is a good ideea or not to return a symbolic value for unknown accounts.

Comment (PetarMax) [reply tests/foundry/test/AccountParamsTest.t.sol:47]: Understood, but won't the test need changing if a symbolic value is returned?

**Review by PetarMax — APPROVED** [review]

**Pull request merged**